src/main.ts

```diff
@@ -20,32 +24,31 @@ export interface LaunchRequestArguments {
 	/** An absolute path to the program to debug. */
 	program: string;
 	/** Automatically stop target after launch. If not specified, target does not stop. */
-	stopOnEntry?: boolean;
+	stopOnEntry ? : boolean;
```

Comment:
Let's keep it with what people does in VS Code code base, use `stopOnEntry?` with no inner spaces